src/traversing.js

```diff
@@ -168,10 +159,18 @@ jQuery.each({
 		return elem.contentDocument || jQuery.merge( [], elem.childNodes );
 	}
 }, function( name, fn ) {
+	var // Sorting (which implicitly de-dupes) is required for non-adjacent traversals
```

Comment:
I'm not sure. We have few examples of commenting the first var in a list of more than one, though only one of them [uses this style](https://github.com/jquery/jquery/blob/4cbf02df84dbcaa44b75a64ed832f7dbff2231dd/src/callbacks.js#L45). I'll change it if that's the consensus.